main.go

```diff
@@ -133,7 +93,8 @@ func main() {
 		length := getMetricDataInputLength(discoveryJob)
 		//S3 can have upto 1 day to day will need to address it in seprate block
```

Comment:
Should we remove the command as you seem to have fixed the s3 problems?

Reply:
I haven't fixed it. I've just added a boolean parameter `IgnoreLength` to a struct, which says that a lenght of this metric can be ignored, cause now not only S3 metrics are updated seldom but also Billing ones

Reply:
Thanks for the explanation.